src/olympia/blocklist/tests/test_utils.py

```diff
@@ -21,7 +21,7 @@ def test_datetime_to_ts():
 
 class TestSaveVersionsToBlocks(TestCase):
     def setUp(self):
```

Comment:
Semi-serious nit. Looking only at the diff here it is, like, totally unclear how these changes to the test satisfactorily test the change in the underlying code. I've looked at the larger file and see that the tests being updated are large and test many things. It's not actually clear to me yet that you have covered all of the potential edge cases (you may have but I'll need to read a lot more code to validate that)

Can I ask for you to instead add a smaller test(s) that covers this specific case:
- ReviewBase.user is task user or fallback to updated_by from the block
- human review is true only if the task user is not the updated_by user.

These 2 test scenarios would clearly and concisely test this behavior, making it easy to have faith in the tests moving forward and east to identify what part of the code might be broken if something changes.

Reply:
https://github.com/mozilla/addons-server/pull/22817/commits/d895354e38615b6aeff213b74966907efc9af8a3